release-notes/8.0/8.0.0/release.json

```diff
@@ -0,0 +1,506 @@
+{
+  "release-date": "2023-11-14",
+  "release-version": "8.0.0",
+  "security": true,
+  "cve.json": "https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/8.0/8.0.0/cve.json",
```

Comment:
I like bools. They work so well with programming language syntax Everything else has worse ergonomics.

Reply:
I think we should continue to include the CVE information in this file. It means near zero cost for people to move to this new format. It also means we don't have to change the code that generates this file.

I see the `cve.json` file as a benefit for people who only want that information.

Reply:
So keeping the list in and linking to it so that people interested would know how to find it? Would be fine with me as the list isn't that big. I thought that it might get bigger once we add more information to each CVE. But even then, it's usually not more then 2-5 CVEs in a single release.

The `8.0/8.0.0/release.json` is one of the new files, so I don't fully understand the part about keeping the generator the same.